### ISSUE TITLE
Plan for docs 2.0

### ISSUE THREAD

**Issue by BTCparadigm** [zkSNACKs/WasabiDoc#1022]
Build in a different [branch](https://github.com/zkSNACKs/WasabiDoc/tree/docs-2.0) without changing categories or breaking internal or external links too much.
NOTE: No need to say 2.0 in new docs.

Update all content except WHY WASABI and BUILDING WASABI.

## USING WASABI

Done:
- [x] 10Commandments #1050
- [x] BackupBestPractices
- [x] BitcoinFullNode #1107 
- [x] ELI5 #1049 
- [x] IndustryStandards #1092 
- [x] LockScreen #1092
- [x] PasswordBestPractices
- [x] PrivacyMode #1086 & #1089
- [x] WalletLoad #1090 
- [x] Testnet #1091
- [x] RestoreElectrum #1105 

ToDo:
- [ ] BIPs
- [ ] BuildSource
- [ ] ChangeCoins
- [ ] CoinJoin #1051 
- [ ] ColdWasabi
- [ ] Daemon
- [ ] DeterministicBuild
- [ ] InstallPackage
- [ ] PasswordFinder
- [ ] PayJoin
- [ ] README
- [ ] RPC
- [ ] Receive #1088
- [ ] Send
- [ ] WalletGeneration - @MarnixCroes 
- [ ] WalletRecovery
- [ ] WasabiSetupVM
- [ ] WasabiSetupWhonix

## FAQ
- [ ] Contribution
- [ ] GeneralBitcoinPrivacy
- [ ] Installation
- [x] Introduction #1109 #1112 #1114 #1116 #1120 #1121
- [ ] UseWasabi #1094 #1095 #1096 #1098 #1100 #1101 #1102 #1124 #1125
- [ ] README
  
## GLOSSARY
- [ ] GeneralBitcoin #1122
- [ ] PrivacyWasabi #1119
- [ ] README
  
<br>

- [ ] Update all pictures

<br>

Review and make sure everything works - @Yahiheb

**Comment by BTCparadigm** [comment]
Tasks broken down in to sections. Pick which ever you like and I'll add a user tag behind the task. 
Boxes are checked when that section is updated, MVP-ready and merged.
If some part should be completely removed, open an issue for that discussion.

**Comment by molnard** [comment]
Look like a plan 👍. It **would be good** (not at any cost) to see the progress. Is it possible to change the Azure deployment, only for this branch to be able to see the current status (we do not need the docs.wasabiwallet.io domain anything is good like a simple GitHub page)? @MaxHillebrand do you know how to do this?

**Comment by MarnixCroes** [comment]
I´ll start with _Why Wasabi_

**Comment by MaxHillebrand** [comment]
sorry, don't know how to do the Azure magic. 
You can however build it locally easily.

**Comment by BTCparadigm** [comment]
Tasklist updated

**Comment by BTCparadigm** [comment]
Thoughts about BackupBestPractices and PasswordBestPractices? Good as is or should it be stripped/simplified?
@MaxHillebrand @yahiheb @MarnixCroes

**Comment by MarnixCroes** [comment]
> Thoughts about [BackupBestPractices.md](https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/using-wasabi/BackupBestPractices.md) and [PasswordBestPractices.md](https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/using-wasabi/PasswordBestPractices.md)? 
> Good as is or should it be stripped/simplified?
> @MaxHillebrand @yahiheb @MarnixCroes 

I'd leave as is. Current is very good and still valid

**Comment by BTCparadigm** [comment]
Questions:

1. In 2.0 we don't have LockScreen function, right? So delete?
2. Current IndustryStandards.md looks weird as is. Delete, keep as is or add stuff?

**Comment by MaxHillebrand** [comment]
> Questions:
> 
> 1. In 2.0 we don't have LockScreen function, right? So delete?
> 2. Current IndustryStandards.md looks weird as is. Delete, keep as is or add stuff?

Delete both.

**Comment by yahiheb** [comment]
> Thoughts about [BackupBestPractices.md](https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/using-wasabi/BackupBestPractices.md) and [PasswordBestPractices.md](https://github.com/zkSNACKs/WasabiDoc/blob/master/docs/using-wasabi/PasswordBestPractices.md)? Good as is or should it be stripped/simplified? @MaxHillebrand @yahiheb @MarnixCroes

I would leave it as it is.

> Questions:
> 
> 1. In 2.0 we don't have LockScreen function, right? So delete?
> 2. Current IndustryStandards.md looks weird as is. Delete, keep as is or add stuff?

Delete both.

**Comment by BTCparadigm** [comment]
As discussed in yesterdays call with @MarnixCroes and @yahiheb, we should worry about picture quality too much yet. Snipping tool pic from 2.0 testnet wallet is ok for now. 
Let's go over all of them once content has been updated and take consistent ones with proper highlights and settings etc.
Separate task created to the end of the list, for making sure pictures are as good as they can be.

**Comment by BTCparadigm** [comment]
@MarnixCroes @yahiheb Good job updating docs to 2.0.

Although, I cant keep updating this issue with all the PR's. 
What's the best way to declutter this issue and stay up to date what's done and what's left?

**Comment by MarnixCroes** [comment]
> @MarnixCroes @yahiheb Good job updating docs to 2.0.
> 
> Although, I cant keep updating this issue with all the PR's. What's the best way to declutter this issue and stay up to date what's done and what's left?

I think linking all PR's is not needed, especially for like FAQ where there are many.
When a section is done, simply marking as complete is ok imo

**Comment by MarnixCroes** [comment]
This is finished :tada: 
I'm not aware of anything not being updated.
If so, please comment so I can fix it.
